src/ontology/oeo.omn

```diff
@@ -26,7 +26,7 @@ Annotations:
     dc:contributor "Izzet Kilicarslan",
     dc:contributor "Janna Hastings",
     dc:contributor "Lara Christmann",
-    dc:contributor "Ludwig Hülk",
+    dc:contributor "Ludwig Hülk (@Ludee)",
```

Comment:
@jannahastings Is this there a better way to reference a Person *and* a handle? Maybe something from the FOAF ontology?

Reply:
That is a good question. I will merge this PR and open a new issue with this question.